File: Common/InMotionGIT.Common/DataType/ResultDataBehavior.cs
using System;
using System.Runtime.Serialization;

namespace InMotionGIT.Common.DataType
{

    [DataContract()]
    [Serializable()]
    public class ResultDataBehavior : ResultBehavior
    {

        [DataMember()]
        public long Count { get; set; }
        [DataMember()]
        public object Data { get; set; }

    }

}

File: Common/InMotionGIT.Common/DataType/ResultDataBehavior.vb
Imports System.Runtime.Serialization

Namespace DataType

    <DataContract()>
    <Serializable()>
    Public Class ResultDataBehavior
        Inherits ResultBehavior

        <DataMember()> Public Property Count As Long
        <DataMember()> Public Property Data As Object

    End Class

End Namespace